tests/fsharp/core/nestedModule/test.fsx

```diff
@@ -0,0 +1,3 @@
+module Top.Square.Middle.Level
+
+let value = 10
```

Comment:
* Add a `module rec` test

* Add a `module` test with a single module defined in the file using `module M = `.